src/main/java/org/galatea/starter/service/IexService.java

```diff
@@ -161,13 +178,13 @@ else if (date != null) {
   public List<IexHistoricalPrices> getHistoricalPriceBySymbolAndDate(
```

Comment:
I think that you can just return a IexHistoricalPrices object instead of a list and simplify your method a little bit.